tests/examples/hello/hello/hello.py

```diff
@@ -0,0 +1,167 @@
+import struct
+from enum import IntEnum
+from typing import Any
+
+import numpy as np
+
+import legate.core.types as types
+from legate.core import Rect, Store, get_legate_runtime
+
+from .library import user_context, user_lib
+
+
+class HelloOpCode(IntEnum):
+    HELLO_WORLD = user_lib.cffi.HELLO_WORLD
+    SUM = user_lib.cffi.SUM
+    SQUARE = user_lib.cffi.SQUARE
+    IOTA = user_lib.cffi.IOTA
+
+
+def print_hello(message: str) -> None:
+    """Create a Legate to task to print a message
+
+    Args:
+        message (str): The message to print
+    """
+    task = user_context.create_auto_task(HelloOpCode.HELLO_WORLD)
+    task.add_scalar_arg(message, types.string)
+    task.execute()
+
+
+def print_hellos(message: str, n: int) -> None:
+    """Create a Legate to task launch to print a message
+       in n replicas of the task
+
+    Args:
+        message (str): The message to print
+        n (int): The number of times to print
+    """
+    launch_domain = Rect(lo=[0], hi=[n], exclusive=True)
```

Comment:
```suggestion
    launch_domain = Rect(lo=[0], hi=[n])
```
`exclusive` is default I think